config/base/params.env

```diff
@@ -1,4 +1,4 @@
-odh-model-controller=quay.io/opendatahub/odh-model-controller:fast
+odh-model-controller=quay.io/rh-ee-amcent/odh-model-controller:latest
```

Comment:
is it the right url?

Reply:
@VedantMahabaleshwarkar recommended I leave it as mine until others have tested it, then revert it back